src/sections/events/PastEvents.js

```diff
@@ -17,9 +19,6 @@ const PastEvents = () => {
         PAST EVENTS
       </Typography>
       <EventCard />
```

Comment:
Suraj sir, Event card component le events as a prop linchha.Should we continue using that or state batai tanne event component ma?

Reply:
What do you mean by state bata tanne? I think if not the whole event itself we should pass at least event id in the prop so that we can filter the required event from the state??